src/web/components/Item/Item.test.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Item from './';
+
+describe('Item', () => {
+  let wrapper;
+  let data;
+  let renderItem;
+
+  beforeEach(() => {
+    data = { id: 'GDJETI12K3J4L', name: 'turtle' };
+    renderItem = item => <div>{item.name}</div>;
```

Comment:
Why to use such a method ?

Reply:
I don't really know. 😄 
I wanted to test, if the function worked well once in the component.
Do I have to try it with `jest. fn()` and see how many times it is called and the parameters passed through the input ?